### PR TITLE
support GPU for tensorflow inception

### PR DESCRIPTION
**Pull request by cheyang**
**What this PR does / why we need it**:

1. Make the parameters Cpu, Memory work as described in the Document
2. Support GPU for TensorFlow Inception Model

**Which issue this PR fixes** *(optional, in `fixes #<issue number>(, fixes #<issue_number>, ...)` format, will close that issue when PR gets merged)*: fixes #

**Special notes for your reviewer**:

### CODE REVIEW

**Comment by cheyang** [comment]
This is for https://github.com/kubernetes/charts/pull/4262 which I closed by accidentally.  sorry.

File: incubator/tensorflow-inception/README.md [hunk 1]
```diff
@@ -31,8 +31,9 @@ The following table lists the configurable parameters of the TensorFlow inceptio
 | `ImageTag`       | Container image tag                | `latest`                                                          |
 | `Replicas`       | k8s deployment replicas            | `1`                                                               |
 | `Component`      | k8s selector key                   | `tensorflow-inception`                                            |
-| `Cpu`            | container requested cpu            | `100m`                                                            |
-| `Memory`         | container requested memory         | `512Mi`                                                           |
+| `Resources.limits.cpu`            | container requested cpu            | `100m`                                                            |
```
Comment (timstoop) [inline incubator/tensorflow-inception/README.md]: If you use just a `resources` here with default `{}`, people will know what to do and understand and you don't have to differentiate in the docs between requests and limits.

**Review by timstoop — CHANGES_REQUESTED** [review]

**Comment by cheyang** [comment]
@timstoop , I removed the description about CPU, memory. Take a look again. Thanks.

**Comment by timstoop** [comment]
Now add a description of the `resources` parameter as well. Just simply something like "Set the resource to be allocated and allowed for the Pods."

As a sidenote, not for you to change in this PR, it would be better to de-capitalize all the variables, as variables in Charts should not be capitalised. If you have time, I'd love to see that done in another PR.

**Comment by cheyang** [comment]
> Now add a description of the resources parameter as well. Just simply something like "Set the resource to be allocated and allowed for the Pods."

Thank you for the suggestions. I've updated.

> As a sidenote, not for you to change in this PR, it would be better to de-capitalize all the variables, as variables in Charts should not be capitalised. If you have time, I'd love to see that done in another PR.

I will do it later, thanks.

**Pull request merged**